canvas_sdk/effects/banner_alert/remove_banner_alert.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from canvas_sdk.effects.base import EffectType, _BaseEffect
+
+
+class RemoveBannerAlert(_BaseEffect):
+    """
+    An Effect that will remove/inactivate a banner alert in Canvas.
+    """
+
+    class Meta:
+        effect_type = EffectType.REMOVE_BANNER_ALERT
+
+    patient_id: str
```

Comment:
I changed this from patient_key (Canvas internal reference) to patient_id, which is consistent with other features plugin developers may interact with. (Example: the FHIR API calls this same value the patient's id)